encounters/README.md

```diff
@@ -0,0 +1,201 @@
+
+- length
+- serviceProvider
+
+## JSON FHIR > XML HL7
+An GP Connect FHIR `Encounter` is mapped to a GP2GP HL7v3 `ehrComposition`.  
+
+| Mapped to (XML HL7 ehrComposition child element) | Mapped from (JSON FHIR / other source )                                                                                   |
+|--------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------|
+| id \[@root]                                      | A random UUID generated by the Adaptor                                                                                    |
+| code \[@code]                                    | `Encounter.type[0].coding[0].code` <sup>3</sup>                                                                           |
+| code \[@displayName]                             | `Encounter.type[0].coding[0].display` <sup>3</sup>                                                                        |
+| code \[@codeSystem]                              | fixed value = `"2.16.840.1.113883.2.1.3.2.4.15"`                                                                          |
+| code / originalText                              | `Encounter.type[0].text` or else `Encounter.type[0].coding[0].display` <sup>4</sup>                                       |
+| statusCode                                       | fixed value =`"COMPLETE"`                                                                                                 |
+| effectiveTime                                    | `Enounter.period.start` and `Encounter.period.end` (if present)                                                           |
+| availabilityTime \[@value]                       | `Encounter.period.start`                                                                                                  |
+| author / agentRef / id \[@root]                  | `Encounter.participant[index].individual` where `Encounter.participant[index].type` contains a `coding.code` of `"REC"`   | 
+| author / time \[@value]                          | `List.date` <sup>5</sup>                                                                                                  |
+| location / locatedEntity / code \[@code]         | fixed value = `"394730007"`                                                                                               |
+| location / locatedEntity / code \[@codeSystem]   | fixed value = `"2.16.840.1.113883.2.1.3.2.4.15"`                                                                          |
+| location / locatedEntity / code \[@displayName]  | fixed value = `"Healthcare related organisation"`                                                                         |
+| location / locatedEntity / name                  | The `name` field of the [Location](../locations/README.md) resource referenced by `Encounter.location`                    |
```

Comment:
I think we're missing a `locatedPlace` element between the `locatedEntity` and `name`